Sources/GoogleAI/Chat.swift

```diff
@@ -67,18 +78,33 @@ public class Chat {
 
   /// See ``sendMessageStream(_:)-4abs3``.
   @available(macOS 12.0, *)
-  public func sendMessageStream(_ parts: PartsRepresentable...)
+  public func sendMessageStream(_ parts: any ThrowingPartsRepresentable...)
     -> AsyncThrowingStream<GenerateContentResponse, Error> {
-    return sendMessageStream([ModelContent(parts: parts)])
+    return try sendMessageStream([ModelContent(parts: parts)])
   }
 
   /// Sends a message using the existing history of this chat as context. If successful, the message
   /// and response will be added to the history. If unsuccessful, history will remain unchanged.
   /// - Parameter content: The new content to send as a single chat message.
   /// - Returns: A stream containing the model's response or an error if an error occurred.
   @available(macOS 12.0, *)
-  public func sendMessageStream(_ content: [ModelContent])
+  public func sendMessageStream(_ contentClosure: @autoclosure () throws -> [ModelContent])
```

Comment:
Same as above - do we have a code snippet that demonstrates the call site for this change?

Reply:
@morganchen12 Correct me if I'm missing something but I think line 83 above is an example usage.

Reply:
Since it's now using @autoclosure, I was expecting to see something similar to the samples in https://www.hackingwithswift.com/example-code/language/what-is-the-autoclosure-attribute

Reply:
Can you provide an example?